.travis.yml

```diff
@@ -303,6 +303,7 @@ matrix:
     - bash <(curl -s https://codecov.io/bash) || echo "Codecov did not collect coverage reports"
   # R
   - language: r
+    r: 3.1
```

Comment:
I don't like to only be testing in very old versions. I understand the need to support it, but this does not protect us from changes in R. 

Plus if we start to play with newer features, carefully protected by conditional compiling or whatever, this will never be tested. 

But I don't know if it's cool to test on various versions of R as we typically do, e.g. https://github.com/tidyverse/dplyr/blob/master/.travis.yml#L10